srcpkgs/git-remote-hg/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'git-remote'
+pkgname=git-remote-hg
+version=1.0.2.1
+revision=1
+build_style=gnu-makefile
+make_install_target="install install-doc"
+hostmakedepends="asciidoc"
+short_desc="Transparent bidirectional bridge between Git and Mercurial for Git"
```

Comment:
I'm not clear on the relationship, but it's the official description, so I guess we should stick with it.